qa/integration-tests/src/test/java/io/camunda/zeebe/it/health/BrokerMonitoringEndpointTest.java

```diff
@@ -54,22 +53,6 @@ public static void tearDownClass() {
     sutBroker.stop();
   }
 
-  @Test
-  public void shouldGetMetrics() {
```

Comment:
:question: Do we want to remove the entire test? Isn't it enough to just adjust the path?

Reply:
I figured it's a built-in Spring Boot endpoint, so no need to test it. We can keep it as a sanity check if you'd like.

Reply:
I think that's fair, I was just not sure why it's removed.